astro/src/content/docs/get-started/core-concepts/users.mdx

```diff
@@ -71,21 +71,25 @@ These are the default fields that are available when creating a user. You can ad
+    If this is checked, FusionAuth will not send a message asking the user to verify their email address or phone number. This is only available if [Email verification](/docs/get-started/core-concepts/tenants#email) or [Phone verification](/docs/get-started/core-concepts/tenants#phone) in the tenant is set up.
+
+    <Aside>
+      If both an email address and phone number are supplied and <InlineField>Skip verification</InlineField> is checked, then verification will be skipped for both the email address and the phone number.
+    </Aside>
   </APIField>
   <APIField name="Email" optional>
-    The email address of the user. This is a required field if <InlineField>Send email to set up password</InlineField> is enabled. It must be unique across all users in the tenant.
+    The email address of the user. This is a required field if <InlineField>Send password setup to</InlineField> is set to `Email`. It must be unique across all users in the tenant.
   </APIField>
-  <APIField name="Username" optional>
-    The username of the user. The username is stored and returned as a case-sensitive value, however, a username is considered unique regardless of the case. This is a required field if no <InlineField>Email</InlineField> is provided.
+  <APIField name="Phone number" optional>
+    The phone number of the user. This is a required field if <InlineField>Send password setup to</InlineField> is set to `Phone`.
   </APIField>
-  <APIField name="Mobile phone" optional>
-    The mobile phone number of the user. This is useful for sending push notifications or SMS messages to the user.
+  <APIField name="Username" optional>
+    The username of the user. The username is stored and returned as a case-sensitive value, however, a username is considered unique regardless of the case. This is a required field if no <InlineField>Email</InlineField> or <InlineField>Phone number</InlineField> is provided.
   </APIField>
-  <APIField name="Send email to set up password" optional>
-    If this is checked, FusionAuth will send an email asking them to set their password.
-    If you also have email verification enabled and do not skip verification with <InlineField>Skip email verification</InlineField> setting, only the setup password email will be sent to the user. Setting up the password using the email sent during this user create operation will verify the user’s email if it is not already verified.
+  <APIField name="Send password setup to" optional>
+    If this is set to `Email` or `Phone`, FusionAuth will send an email or SMS message asking the user to set their password.
+    If you also have verification enabled and do not skip verification with <InlineField>Skip verification</InlineField> setting, only the setup password message will be sent to the user. Setting up the password using the message sent during this user create operation will verify the user’s email address or phone number if it is not already verified.
```

Comment:
I believe that a single newline without a blank line in there will render these as the same paragraph. I assume you wanted a separate paragraph?

I thought today's code only verified their email address if implicit verification is enabled. Is that changing?

We should say "set up password message" or "password setup message."

Reply:
I didn't give enough thought to spacing. Pushed a fix with an aside that seems to look good.

As to your question about implicit, no that's not changing. I tried to improve on the existing docs (they don't mention this either) with the push I just made. Not sure how to make it pretty, but I think listing things without a bulleted list is harder to follow.